### PR TITLE
Revert "Revert "Feature/360 model extract tab updates""

### PR DESCRIPTION
**Pull request by gidsg**
Restore these changes.
Reverts communitiesuk/funding-service-design-post-award-data-store#170

### CODE REVIEW

File: core/serialization/download_xlsx_serializer.py [hunk 1]
```diff
@@ -38,8 +38,8 @@ def serialize_xlsx_data(programmes, programme_outcomes, projects, project_outcom
     programme_outcomes = [
         OutcomeDataSerializer(
             outcome_data,
-            programme_name_mapping[outcome_data.programme.programme_id],
-            organisation_mapping[outcome_data.programme.programme_id],
+            programme_name_mapping.get(outcome_data.programme.programme_id, None),
```
Comment (gidsg) [inline core/serialization/download_xlsx_serializer.py:41]: Do we need the `None` second argument? Is that not the default behaviour of `.get`?

Comment (nsnape) [reply core/serialization/download_xlsx_serializer.py:41]: No we don't, I forgot to remove after checking

File: tests/test_serialization.py [hunk 2]
```diff
@@ -1,21 +1,62 @@
...
+    assert serialized_data.get("ProjectProgress")
+    assert serialized_data.get("FundingQuestions")
+    assert serialized_data.get("Funding")
+    assert serialized_data.get("FundingComments")
+    assert serialized_data.get("PrivateInvestments")
+    assert serialized_data.get("OutputsRef")
+    assert serialized_data.get("OutputData")
+    assert serialized_data.get("OutcomeRef")
+    assert serialized_data.get("OutcomeData")
+    assert serialized_data.get("RiskRegister")
```
Comment (gidsg) [inline tests/test_serialization.py:45]: Are we expecting `ExtractedPostcodes` in the XSLX serialisation? I think we should have it if possible.

**Review by nsnape — APPROVED** [review]
approving for Gideon as he can't do it as creator

File: tests/test_serialization.py (hunk 2 above)
Comment (gidsg) [reply tests/test_serialization.py:45]: Scrap that I found it now in the ProjectDetails tab.

**Pull request merged**